modules/release-notes/pages/0.7.0-rn.adoc

```diff
@@ -50,7 +50,7 @@ This change allows you to specify the format for the return result for `+dfx can
 
 * The default network for projects is now `ic`.
 +
-The default network is now the mainnet `+ic` unless otherwise specified.  Additionally, `+--network ic+` now points to the mainnet IC (Sodium has been deprecated.)
+The default network is now the mainnet `+ic+` unless otherwise specified.  Additionally, `+--network ic+` now points to the mainnet IC (Sodium has been deprecated.)
```

Comment:
"IC" maybe should be an `.adoc` variable expanding to "Internet Computer".

Reply:
Yes should be {IC}

Reply:
```suggestion
The default network is now the mainnet `+ic+` unless otherwise specified.  Additionally, `+--network ic+` now points to the mainnet {IC} (Sodium has been deprecated.)
```